csharp/device/Microsoft.Azure.Devices.Client/Common/SHA.cs

```diff
@@ -0,0 +1,211 @@
+// 
+//  Code by Elze Kool
```

Comment:
@josesimoes @elzekool Finally got an answer! Apparently this needs to be explicitely added in a header/comments section at the beginning of the file: “Original author Elze Kool has dedicated this code (SHA.cs) to the public domain.” Thanks!